extension/src/panel.ts

```diff
@@ -239,9 +256,8 @@ export class HistoryDebuggerPanel {
             break
           }
           case "startFromHistory": {
-            const buffer = Buffer.from(e.buffer)
             // TODO: support binary history too
-            const history = historyFromJSON(JSON.parse(buffer.toString()))
+            const history = historyFromJSON(e.history)
```

Comment:
We need to eventually support loading binary too, as long as that works eventually, it should be fine to make this change.

Reply:
Can you tell me a bit more about what binary history might look like? If we need to support formats other than JSON then we would need to revisit this in order to get it working.

Reply:
We can handle this later, please open an issue to track this.

Reply:
Added https://github.com/temporalio/vscode-debugger-plugin/issues/23 ✅